docs(range tokens): update documentation

abg4 opened the pull request.

smb2796 commented on docs/range-tokens/deployment-tutorial.md:
```diff
@@ -0,0 +1,117 @@
+sidebar_label: Deployment Tutorial
+---
+
+The process to launch and manage a range token on UMA is surprisingly simple. This tutorial will show you how to deploy a range token using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository. By following these instructions, you will be able to launch a custom LSP on Kovan testnet and Ethereum mainnet.
```
```suggestion
The process to launch and manage a range token on UMA is surprisingly simple. This tutorial will show you how to deploy a range token using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository.
```

abg4 replied:
Updated and made sure the change was also included in the success token section.

smb2796 commented on docs/range-tokens/deployment-tutorial.md:
```diff
@@ -0,0 +1,117 @@
+
+To add a new price identifier or collateral currency to Kovan, please contact the UMA team in Discord.
+
+## Clone the repo
```
What do you think about adding a separate launch-lsp prereqs page, since this content will be redundant in all of the deployment tutorials?

abg4 replied:
I agree and I implemented the change we talked about with creating a separate section in Developers > Quick Start > LSP Prerequisites

smb2796 commented on docs/range-tokens/deployment-tutorial.md:
```diff
@@ -0,0 +1,117 @@
+
+### Mandatory deployment parameters
+
+- `gasprice:` The gas price used for your contract deployment.
```
I think it'd be cool if we gave more context to this stuff. Right now it basically reads as a general lsp launch tutorial. Thoughts on adding a range token example? Could maybe use UMA's range token that recently expired, which would give some real world context to this.

Then you could talk about the fact that you need to use UMA as collateral and UMA/USD as the price identifier. Could also give a rough example of pricing after deployment. And also give some context to what setting the bounds means.

abg4 replied:
Updated to include more context. Let me know if this isn't what you were looking for but kept it consistent with using the UMA range token as an example across all sections.

smb2796 commented on docs/range-tokens/summary.md:
```diff
@@ -0,0 +1,40 @@
+
+## Range Token Example
+
+DAOs have become massive with some treasuries valued in the billions of dollars. However, the majority of these assets are in the native token of the project with only a fraction in liquid assets or stable coins. An example of a project using range tokens would be if the Uniswap treasury decided to use $UNI tokens as collateral to borrow USDC. The treasury could create the following terms for the ...
```
```suggestion
DAOs have become massive with some treasuries valued in the billions of dollars. However, the majority of these assets are in the native token of the project with only a fraction in liquid assets or stable coins. An example of a project using range tokens would be if the UMA treasury decided to use $UMA tokens as collateral to borrow USDC. The treasury could create the following terms for the range token contract:
```

I know this is all grandfathered in from the previous tutorial, but can we replace all of the UNI examples with UMA?

abg4 replied:
Updated all UNI examples to UMA.

smb2796 commented on docs/range-tokens/usage-tutorial.md:
```diff
@@ -0,0 +1,36 @@
+
+Once the price is accepted, the `expiryPrice` returned by the Optimistic Oracle is used by the [RangeBond](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/RangeBondLongShortPairFinancialProductLibrary.sol) financial product library to calculate `expiryPercentLong`. `expiryPrice` and `expir ...
+
+`settle` can now be called which uses `ExpiryPercentLong` to determine the redemption rate between the long and short tokens by returning a number between 0 and 1, where 0 sends all collateral to the short tokens and 1 sends all collateral to the long tokens. The collateral returned is the sum of the two payouts and both the long and short tokens are burned.
```
```suggestion
`settle` can now be called which uses `ExpiryPercentLong` to determine the redemption rate between the long and short tokens by returning a number between 0 and 1, where a value of 0 allocates all collateral to the short tokens and a value of 1 allocates all collateral to the long tokens. The collateral returned is the sum of the two payouts and both the long and short tokens are burned.
```

I think this could use a numerical example. User A has X short tokens and Y long tokens and receives Q collateral back etc.

abg4 replied:
I included a numerical example using the UMA range token in the section below. Let me know your thoughts.

smb2796 commented on docs/range-tokens/usage-tutorial.md:
```diff
@@ -0,0 +1,36 @@
+sidebar_label: Usage Tutorial
+---
+
+Now that you have a deployed LSP contract, this tutorial will walk you through the four ways to interact with the contract and your range tokens.
```
I think this page is good, but might be more understandable with an actual concrete example (similar recommendation as the one for the deployment tutorial). It'd be cool to use the same contract example for both the deployment and usage tutorial

abg4 replied:
Included examples using the UMA range token. Let me know if there is anywhere you see an opportunity to add an example that I missed.

smb2796 commented:
Let's remove the featured products sidebar item now that the content is in the other sections.

smb2796 commented on docs/developers/lsp-prereqs.md:
```diff
@@ -0,0 +1,41 @@
+---
```
I think this page works better in the developers / LongShortPair(LSP) section. Wdyt?

abg4 replied:
Yea, I agree that makes more sense. Updated to be before the deploy-an-lsp section.

smb2796 commented on docs/kpi-options/deployment-tutorial.md:
```diff
@@ -0,0 +1,130 @@
+
+The following is an overview of the deployment steps below.
+
+1. LSP Prerequisites
```
This is a title, while the others are actions. Maybe just `Setup LSP Deployment Prerequisites`

abg4 replied:
Updated for range tokens, success tokens, and KPI options

abg4 commented:
Removed the Featured Products section. Also, removed the:
- mint-locally page
- developer mining section
- fixed typo that was called out in Discord

Wanted to get a few of these changes in as the other developer section changes (updating graphics/dev guide) aren't ready yet.
